manoseimas/mps_v2/models.py

```diff
@@ -96,7 +97,7 @@ def __unicode__(self):
 class GroupMembership(CrawledItem):
     member = models.ForeignKey(ParliamentMember)
     group = models.ForeignKey(Group)
-    position = models.CharField(max_length=64)
+    position = models.CharField(max_length=128)
     since = models.DateField(blank=True, null=True)
     until = models.DateField(blank=True, null=True)
 
```

Comment:
Storing *all* models in `core` kind of kills modularity. What you can do is to really create a `core` set of models all your apps can depend on.

But still I have mixed opinion about this. On one hand it is good to have this set of core models for your project and have all other apps depend on them instead of willy-nilly inter-module dependencies. On the other hand, your project becomes less modular since you have to provide the same `core` for each of your apps (modules) to work and it is very hard to avoid inter-module dependencies.